examples/one-rate-plan/README.md

```diff
@@ -0,0 +1,13 @@
+# Complete Example (multiple COS Buckets with retention, encryption, tracking and monitoring enabled)
```

Comment:
The description title and contents need to be aligned with the resources in the main.tf